packages/vats/src/provisionPool.js

```diff
@@ -70,20 +67,16 @@ const makeBridgeProvisionTool = (sendInitialPayment, publishMetrics) => {
         await sendInitialPayment(address, bank);
         // only proceed  if we can provide funds
 
-        const { nameHub, myAddressNameAdmin } =
-          makeMyAddressNameAdminKit(address);
-
-        await Promise.all([
-          E(namesByAddressAdmin).update(address, nameHub, myAddressNameAdmin),
-          E(walletFactory).provideSmartWallet(
-            address,
-            bank,
-            myAddressNameAdmin,
-          ),
-        ]);
-        provisionedCount += 1n;
+        const [_, created] = await E(walletFactory).provideSmartWallet(
+          address,
+          bank,
+          namesByAddressAdmin,
+        );
+        if (created) {
+          provisionedCount += 1n;
```

Comment:
"provision" is being used for the one-time event and the recurring "provideSmartWallet" that is the provider pattern (idempotent). It may help to distinguish these terms more.

e.g. here `totalCreated`.

Reply:
That would be a contract API change, which seems out of scope of this issue/PR and fairly risky at this point in the release process.

I suppose refining the meaning of `provisionedCount` is also an API change, but it seems like a much smaller one.

Reply:
I just meant for the internal value, not the external contract which would still be `walletsProvisioned`.